xradar/util.py

```diff
@@ -648,3 +650,55 @@ def _map_over_sweeps(*args, **kwargs):
         return xr.map_over_datasets(functools.partial(_func, **kwargs), *args)
 
     return _map_over_sweeps
+
+
+def dataset_vars(sweep):
```

Comment:
I'm worrying a bit about the naming. This is too close to the xarray nomenclature (`Dataset.data_vars`). I do not have an ad-hoc suggestion, maybe prepending it with something suitable?

Reply:
I suggest "get_".